lib/package.rb

```diff
@@ -164,13 +164,19 @@ def self.system(*args)
       { exception: true }
     end
 
-    # add -j arg to build commands
+    # Add -j arg to build commands.
     cmd_args.sub(/\b(?<=make)(?=\b)/, " -j#{CREW_NPROC}") unless cmd_args =~ /-j\s*\d+/
+    # Escape special bash characters.
+    cmd_args.gsub!('"','\\\\"')
+    cmd_args.gsub!('$','\\\\$')
+    cmd_args.gsub!('`','\\\\`')
 
     begin
       # use bash instead of /bin/sh
       @system_args = ''
       [@system_args, "bash -e -c \"#{cmd_args}\""].reduce(&:concat)
```

Comment:
Passing the parameters separately to avoid calling `/bin/sh` before calling `bash` (then they don't need to be escaped)?

Reply:
(and use `env` to pass the env variables to bash)